src/components/data-table/ha-data-table.ts

```diff
@@ -89,50 +99,52 @@ export class HaDataTable extends BaseElement {
   @property({ type: String }) private _filter = "";
   @property({ type: String }) private _sortColumn?: string;
   @property({ type: String }) private _sortDirection: SortingDirection = null;
+  private _filteredData: DataTabelRowData[] = [];
+
+  private _debounceSearch = debounce(
```

Comment:
I'd love to do this without debounce and the 200ms delay, but couldn't get it to work, if you could guide me :-)

Reply:
Do we still need this now we also have the other debounce ?